src/block/block_open.c

```diff
@@ -132,6 +132,7 @@ __wt_block_close(WT_SESSION_IMPL *session, WT_BLOCK *block)
     }
 
     __wt_free(session, block->name);
+    __wt_spin_destroy(session, &block->cache_lock);
```

Comment:
This line feels strange in this location splitting the two `wt_free` calls. Maybe alphabetize the function and put below the 2nd free, or alphabetize the field and move above the first free.

Reply:
This is in `WT_BLOCK` structure field order.